client/app/scripts/superdesk-archive/tests/archive_spec.js

```diff
@@ -22,7 +22,7 @@ describe('content', function() {
 
     describe('multi service', function() {
         it('can reset on route change', inject(function(multi, $rootScope) {
-            multi.toggle({_id: 1});
+            multi.toggle({_id: 1, selected: true});
```

Comment:
The item.selected is set as a model for selection so it will be changed directly when the selection change is performed.